src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -109,11 +109,42 @@ class RsErrorAnnotator : AnnotatorBase(), HighlightRangeExtension {
             override fun visitPatTupleStruct(o: RsPatTupleStruct) = checkRsPatTupleStruct(rsHolder, o)
             override fun visitPatTup(o: RsPatTup) = checkRsPatTup(rsHolder, o)
             override fun visitStructLiteralField(o: RsStructLiteralField) = checkReferenceIsPublic(o, o, rsHolder)
+            override fun visitMetaItem(o: RsMetaItem) = checkMetaItem(rsHolder, o)
         }
 
         element.accept(visitor)
     }
 
+    private fun checkMetaItem(holder: RsAnnotationHolder, metaItem: RsMetaItem) {
+        val args = metaItem.metaItemArgs
+        val name = metaItem.name
+        if (metaItem.isRootMetaItem() && args != null && name in listOf("cfg", "cfg_attr")) {
+            val item = args.metaItemList.getOrNull(0) ?: return
+            checkCfgPredicate(holder, item)
+        }
+    }
+
+    private fun checkCfgPredicate(holder: RsAnnotationHolder, item: RsMetaItem) {
+        val itemName = item.name ?: return
+        val args = item.metaItemArgs ?: return
+        when (itemName) {
+            "all", "any" -> args.metaItemList.forEach { checkCfgPredicate(holder, it) }
+            "not" -> {
+                val parameter = args.metaItemList.getOrNull(0) ?: return
+                checkCfgPredicate(holder, parameter)
+            }
+            "version" -> {
+                val path = item.path ?: return
+                RsDiagnostic.ExperimentalFeature(path, null, "`cfg(version)` is experimental and subject to change",
```

Comment:
Let's just do nothing in the case of `version` attribute. Currently, this code produces false-positive error even if you have `#![feature(cfg_version)]` and nightly toolchain.
![image](https://user-images.githubusercontent.com/2539310/115828919-fc0edb80-a416-11eb-9c00-b4234bf6cb20.png)

If you would like to support it as well, let's do it in separate PR